### ISSUE TITLE
Support for Fujitsu E754

### ISSUE THREAD

**Issue by coudim** [he1per/psmouse-dkms-alpsv7#3]
Hi all,
First of all: great work :) really ppreciated.

Unfortunately, new Fujitsu laptops like Lifebook E754, need additional IDs in alps.c:
Unknown ALPS touchpad: E7=10 00 64, EC=10 00 64
Adding { 0x10, 0x00, 0x64 } in alps_model_data[] seems to do the trick, but probably additional configuration i needed. 

thanks again :+1:

**Comment by he1per** [comment]
Try the new version, includes better error messages and new patches for the mouse driver.

**Comment by stimpy2k7** [comment]
Hi,
thanks for your great work! You helped me a lot with your driver and patches.

But I had big problems to get them working with my Fujitsu E754 (no working keyboard and touchpad at all under debian and ubuntu). Perhaps the following info help other people who have the same issues with the E754.

I used your psmouse-dkms-alpsv7 and added the following info to the alps.c:

{ { 0x10, 0x00, 0x64 }, 0x00, ALPS_PROTO_V2, 0xf8, 0xf8, ALPS_PASS | ALPS_DUALPOINT | ALPS_PS2_INTERLEAVED }                /\* Fujitsu e754 */

after installing your package I just had to create the file

"/etc/modprobe.d/psmouse.conf"

with the following entry

"options psmouse proto=any"

At last I had to run "update-initramfs -u".

Now it is working completly perfekt with two-finger scrolling and stills works after reboot (using Xubuntu) ;)

Thanks again!
